### PR TITLE
Use cr.flyte.org to fetch flyte containers

### PR DESCRIPTION
**Pull request by aviaviavi**
This change updates ghcr.io container references to instead use flyte's new cr.flyte.org domain. All generated yaml files have been updated as well. Please let me know if I missed anything, changed something that shouldn't be, etc. Thanks!

### CODE REVIEW

File: script/generate_helm.sh [hunk 1]
```diff
@@ -21,7 +21,7 @@ then
     GO111MODULE=on go get github.com/norwoodj/helm-docs/cmd/helm-docs
 fi
 
-~/go/bin/helm-docs -t ${DIR}/../helm/README.md.gotmpl ${DIR}/../helm/
+${GOPATH}/bin/helm-docs -t ${DIR}/../helm/README.md.gotmpl ${DIR}/../helm/
```
Comment (aviaviavi) [inline script/generate_helm.sh:24]: I ran into issues while running `make helm` as my GOPATH is somewhere else, so I bundled this change which seems like an improvement but let me know if you'd prefer this be reverted or put in its own PR.

File: deployment/test/flyte_generated.yaml [hunk 2]
```diff
@@ -474,7 +474,7 @@ data:
 
 kind: ConfigMap
 metadata:
-  name: flyte-propeller-config-dtdt5gbgg2
+  name: flyte-propeller-config-d9cm925f8m
```
Comment (aviaviavi) [inline deployment/test/flyte_generated.yaml:477]: I think these sums are supposed to get updated when generating these files, but let me know if this looks incorrect

**Comment by EngHabu** [comment]
Great! Thank you for taking care of this. 
Mind clicking on "Details" for DCO action and applying the actions it recommends to sign your commits?

**Comment by aviaviavi** [comment]
On it!

**Review by EngHabu — APPROVED** [review]
Thank you for doing this

**Review by EngHabu — APPROVED** [review]
